File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> check;
        for (int i = 0; i < nums.size(); i++){
            int diff = target - nums[i];
            if(check.count(diff)){
                return {check[diff], i};
            }
            else{
                check[nums[i]] = i;
            }
        }
        return {};
    }
};

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        count = {}
        res = []
        target = math.floor(len(nums)/3)

        for i in nums:
            count[i] = 1 + count.get(i, 0)

            if count[i] > target and i not in res:
                res.append(i)
        
        return res